account_invoice_constraint_chronology/model/account_move.py

```diff
@@ -67,12 +67,14 @@ def _raise_newer_conflicting_invoices(self):
     def write(self, vals):
         if vals.get("state") != "posted":
             return super().write(vals)
-
+        previously_validated = self.filtered(lambda m: m.name and m.name != "/")
         newly_posted = self.filtered(lambda move: move.state != "posted")
         res = super().write(vals)
         for move in newly_posted & self.filtered("journal_id.check_chronology"):
             if self.search(move._get_older_conflicting_invoices_domain(), limit=1):
                 move._raise_older_conflicting_invoices()
+            if move in previously_validated:
```

Comment:
Do directly the comparison here and remove the previous `filtered` for performance:
```suggestion
            if (move.name or "/") != "/":
```

Reply:
Ok :)